tools/github/scripts/Sync-AzsdkLabels.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding(DefaultParameterSetName = 'RepositoryFile', SupportsShouldProcess = $true)]
+param (
+    [Parameter(Position = 0, Mandatory = $true)]
+    [ValidatePattern('^\w+/[\w-]+$')]
+    [string] $SourceRepository,
+
+    [Parameter(ParameterSetName = 'RepositoryFile')]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string] $RepositoryFilePath = "$PSScriptRoot/../repositories.txt",
+
+    [Parameter(ParameterSetName = 'Repositories')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Repositories = @(
+        'Azure/azure-sdk-for-cpp'
+        'Azure/azure-sdk-for-go'
+        'Azure/azure-sdk-for-java'
+        'Azure/azure-sdk-for-js'
+        'Azure/azure-sdk-for-net'
+        'Azure/azure-sdk-for-python'
+        'Azure/azure-sdk-tools'
+    ),
+
+    [Parameter(ParameterSetName = 'Languages')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Languages = @('cpp', 'go', 'java', 'js', 'net', 'python'),
+
+    [Parameter()]
+    [switch] $Force
+)
+
+if (!(Get-Command -Type Application gh -ErrorAction Ignore)) {
+    throw 'You must first install the GitHub CLI: https://github.com/cli/cli/tree/trunk#installation'
+}
+
+if ($PSCmdlet.ParameterSetName -eq 'Languages') {
+    $Repositories = foreach ($lang in $Languages) {
+        "Azure/azure-sdk-for-$lang"
+    }
+}
+
+if ($PSCmdlet.ParameterSetName -eq 'RepositoryFile') {
+    $Repositories = Get-Content $RepositoryFilePath
+}
+
+# Filter out the source repository.
+$Repositories = $Repositories.Where({$_ -ne $SourceRepository})
+
+foreach ($repo in $Repositories) {
+    if ($Force -or $PSCmdlet.ShouldProcess(
```

Comment:
@heaths we do have `NeedsARMTriage` custom label in `azure-sdk-tools` repo. Please do not delete it :D

Reply:
`gh label clone` is non-destructive. It does upserts only.

Reply:
If you're interested in the code, see https://github.com/cli/cli/blob/b292dc43b25183024fb12b78459e0d940f524027/pkg/cmd/label/clone.go and https://github.com/cli/cli/blob/b292dc43b25183024fb12b78459e0d940f524027/pkg/cmd/label/create.go#L128